src/Tribe/Admin/Event_Meta_Box.php

```diff
@@ -183,6 +184,70 @@ protected function set_end_date_time() {
 		$this->vars['EventEndTime'] = ( isset( $end_time ) && $end_time ? $end_time : null );
 
 		$this->vars['end_timepicker_step'] = $this->get_timepicker_step( 'end' );
+		$this->vars['end_timepicker_default'] = $this->get_timepicker_default( 'end' );
+	}
+
+	/**
+	 * Check if the Event is an Auto-Draft
+	 *
+	 * @since 4.4
+	 *
+	 * @return bool
+	 */
+	public function is_auto_draft() {
+		if ( ! $this->event instanceof WP_Post ) {
+			return true;
+		}
+
+		// Fetch Status to check what we need to do
+		$status = get_post_status( $this->event->ID );
+
+		if ( ! is_string( $status ) || 'auto-draft' !== $status ) {
+			return true;
```

Comment:
Seems like this will have us returning true if the status is `'publish'` and returning false if the status is `'auto-draft'` - we should either change the if condition or else flip the return statements.